src/Html/Events/Extra/Pointer.elm

```diff
@@ -52,8 +52,15 @@ And to know if the shift key was pressed:
     isShiftKeyPressed event =
         event.pointer.key.shift
 
+*Note about `movementX/Y`: although official docs say that support for
```

Comment:
```suggestion
*Note about `movementX/Y`: official docs say that support for
```
Also we can drop the bold emphasis

Reply:
well not **bold** emphasis, just *italic* emphasis, but yeah we can drop the emphasis

Reply:
Hm, my intention was to make this part more explicit, but on the other hand it's just the same part of the docs. I'll commit it